{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,111 +1,160 @@
 # {{cookiecutter.channel_name}} Chef
-
-Kolibri is an open source educational platform to distribute content to areas with
-little or no internet connectivity. Educational content is created and edited on [Kolibri Studio](https://studio.learningequality.org),
-which is a platform for organizing content to import from the Kolibri applications. The purpose
-of this project is to create a *chef*, or a program that scrapes a content source and puts it
-into a format that can be imported into Kolibri Studio. {% if cookiecutter.chef_template == 'Sous Chef' -%}This project will read a
-given source's content, parse it, and organize that the content files into folder and subfolders,
-with the metadata provided as CSV files, the whole thing inside a zip archive.
-The zip archive can then be imported into Kolibri Studio using a LineCook script.
-(examples souschef scripts can be in the `examples` directory. {%- endif %}
+This repository contains the code for creating the {{cookiecutter.channel_name}}
+channel in Kolibri. The script `sushichef.py` uploads the content to Kolibri Studio,
```

Comment:
I'd suggest we change this to "The script `sushichef.py` downloads the source content, converts it for use in Kolibri, and then uploads it into Kolibri Studio, (... the rest ...)"